markdown/3.0/en/ff-searchbox.md

```diff
@@ -24,58 +24,73 @@ The following code-example shows the aforementioned configuration.
 ```
 For more information, see the [API reference](/api/ff-searchbox#tab=api).
 
-## Extending the input
-The `ff-searchbox` extends a normal `<input>`. Simply add the attribute
-`is="ff-searchbox"` to a standard `<input>` element. On ENTER a search
-query is fired with the current value.
+## Wrapping input element with ff-searchbox
+The `ff-searchbox` wraps a regular HTML `<input>` element. Simply place
+`<input>` between `<ff-searchbox>` tags. On ENTER a search query is fired
+with the current input value.
 ```html
-<input is="ff-searchbox" />
+<ff-searchbox>
+    <input />
+</ff-searchbox>
 ```
 
 ### Placeholder
-Because the `ff-searchbox` is an extension to a normal `<input>` you can
-use all the standard attributes of `<input>` such as `placeholder`.
+Because the `ff-searchbox` does not limit features o HTML it wraps, 
```

Comment:
o**f** HTML

But maybe just remove the first part, because why shouldn't it be that way?

On second part, maybe even remove the full paragraph? It is standard HTML, isn't it?

Reply:
We both know it's standard HTML, but end user does not necessarily does. For some reason this section was there and I'd rather keep it.

Reply:
> o**f** HTML

sorry, seems that this was a to lazy hint from me. Leaving the sentence there is fine with me, but please spend a **f** to  the lonely _o_ ;)